### DISCUSSION TITLE
RTL support

### DISCUSSION THREAD

**Discussion by ciampo** [vanilla-extract-css/vanilla-extract#503] [Q&A] [answered]
Heya 👋 

Does `vanilla-extract` has any out-of-the-box support for RTL styles, or does it plan to offer it in the future?

Thank you and keep up the good work!

**Comment by mattcompiles** [comment, marked as answer]
Hey 👋 

The vanilla-extract API is designed to be as close to regular CSS as possible, but allowing you to build abstractions on top just like you would with TypeScript/JavaScript. There's no plans to to ship any official APIs to handle RTL, but you should be able to build some utilities in your own project to centralise the problem (you could even open source it). That or build it into your [Sprinkles](https://vanilla-extract.style/documentation/sprinkles-api/) utilities, whatever you feel models the problem best for your project. 

You can also use something like postcss plugins as a post-processing step within your bundler.
